### PR TITLE
Remove old versions of direct ethernet

### PR DESCRIPTION
**Pull request by ejeffrey**
These are out of date.

### CODE REVIEW

**Comment by jwenner** [comment]
I strongly disagree with this PR. This is the last publicly released version of the Direct Ethernet server. If we remove this, how are groups outside of ours supposed to talk with their DAC/ADC boards?

**Comment by ejeffrey** [comment]
If the up to date code is not here, they need to be getting it from us directly.  Or we should move the code from pyle back here.

It will still, of course, be available in the history, but by publishing obsolete code doesn't help anyone.

**Comment by maffoo** [comment]
@jwenner, also, the binaries that we've built are published on bintray (https://bintray.com/labrad/generic/direct-ethernet) and at least the last release of the direct ethernet server was tagged as [direct-ethernet-v1.2.3](https://github.com/martinisgroup/servers/tree/direct-ethernet-v1.2.3) so it is here in the repo and always possible to go back. So nothing is going away for people who need it, this would just be cleaning up the tip of tree to only contain stuff that is being actively developed.

I do agree that we need to discuss the best way to deal with things here, however, because this repository is kind of a hodgepodge, and different people are probably using the code in different ways.

**Comment by jwenner** [comment]
Please see rationale for having the updated code in a private repo here: martinisgroup/pyle#1073.

And as an example of another such group using the qubit sequencer code, Haohua's group (see #288), and Robert's group uses the FPGA server and so direct ethernet server.

Now, what I would support is putting a note in the readmes saying that these servers are no longer being updated and to contact us for the latest versions.

**Comment by ejeffrey** [comment]
OK, I am going to leave the fpga server and calibration server for now (but remember, the calibrations server is broken!).

The direct ethernet server and qubit server I am going to remove from master, but add a readme with instructions pointing to both the bintray release and the release tag for the source repo.

**Comment by ejeffrey** [comment]
OK, notes added with both bintray and github links.

I really don't want someone checking this out and thinking it is the latest repo, and by referring to a tag, that will prevent anyone from making modifications and trying to submit a PR which we can't accept.

File: Readme_Direct_Ethernet.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+The direct ethernet is no longer being actively developed here. To
+avoid confusion it has been removed from master.
+
+The latest binary release (1.2.3) is available via bintray:
+
+https://bintray.com/labrad/generic/direct-ethernet
+
+The source code is available in a release tag of the git repository:
+
+https://github.com/martinisgroup/servers/tree/direct-ethernet-v1.2.3
```
Comment (maffoo) [inline Readme_Direct_Ethernet.txt]: I would suggest putting all these notes in the main `README.md` so this is shows up for people browsing the repo in the web interface.

File: Readme_Direct_Ethernet.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+The direct ethernet is no longer being actively developed here. To
+avoid confusion it has been removed from master.
+
+The latest binary release (1.2.3) is available via bintray:
+
+https://bintray.com/labrad/generic/direct-ethernet
+
+The source code is available in a release tag of the git repository:
```
Comment (maffoo) [inline Readme_Direct_Ethernet.txt]: Here, can just say that the latest release is at tag `direct-ethernet-v1.2.3`. Can give the github link as well, but people using git command line tools just need to know the tag name.

File: Readme_Direct_Ethernet.txt (hunk 1 above)
Comment (ejeffrey) [inline Readme_Direct_Ethernet.txt]: I will mention it in README.md, but someone who checks out the repo and is looking for the files may not see it there.

File: DirectEthernet/README.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+The direct ethernet is no longer being actively developed here. To
```
Comment (maffoo) [inline DirectEthernet/README.txt]: Maybe makes these `.md` files. Github renders them nicely (with links, etc) and they're still perfectly readable as plain text.

**Comment by ejeffrey** [comment]
OK, converted to .md files

**Comment by maffoo** [comment]
LGTM

**Pull request merged**